.github/workflows/ci.yml

```
@@ -110,6 +112,23 @@ jobs:
         run: yarn lerna run test-unit-ibmi --scope=@sequelize/core
       - name: Unit tests (snowflake)
         run: yarn lerna run test-unit-snowflake --scope=@sequelize/core
+  unit-test-win:
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: [16, 20]
+    name: Simple Windows Unit test all dialects (Node ${{ matrix.node-version }})
```

```suggestion
    name: Build and test on Windows (Node ${{ matrix.node-version }})
```
This CI job could indeed be combined so we only do this once